workspaces/liferay-sample-workspace/client-extensions/liferay-sample-custom-checkout-step/src/main/resources/application-default.properties

```diff
@@ -0,0 +1,20 @@
+#
+# LXC
+#
+
+com.liferay.lxc.dxp.domains=localhost:8080
```

Comment:
These are only valid when working on local environments. I don't think they have to be merged. Other CX don't have them.